trainings/sourcegraph-101/talk-track.md

```diff
@@ -0,0 +1,327 @@
+# Sourcegraph 101
+
+**Topic:** This training is focused on end users, and serves as a start-from-zero training for new end users. Admin tasks and more advanced search functionality are covered in other trainings. The training is broken into 5 units, but you don't need to call the units out to the customer—they serve more as a way to structure the training for yourself.
+
+## Intro
+
+Hi! Today I'm going to be walking you thorugh what we call Sourcegraph 101—it should take you from 0 to being able to use Sourcegraph confidently. We of course don't have time for a dive into all of what Sourcegraph can do today, so we'll be running a Sourcegraph 102 training in our next session to talk about advanced features. That said, throughout today, please ask questions! You can take yourself off mute or drop it into the chat, and my colleague will read them out for us when we come to a pause. Even if I can't answer the question today, I'll connect you to someone who can.
```

Comment:
This is intended as a talk track, thus the script format. There will be separate customer-facing versions of the content intended for slotting into a customer LMS, so I don't think it makes sense to write this as if it's customer facing. I chose this format primarily because of feedback I've heard from CEs about challenges with putting that content together into their own script. 

@amenne , do you have thoughts?

Reply:
I think I answered above but I see value in both immediately: script and lesson plan. Over time as this training piece moves entirely away from CEs and over to dedicated trainers we won't need this but many CEs have still yet to go through these things for the first time so they might find more guidance helpful (we don't want them simply reading this verbatim -- we want them to make it their own natural voice).

We gotta think about this as crawl, walk, run. I'd argue we're at the baby can roll over stage even. Customer-facing / self-service is 10000% the direction we wanna get to and I recognize that will look differently, too.

Reply:
this PR represents tummy time.

Reply:
Given this, seems like the direction is to keep the content as-is and push a lesson plan separately (same directory, separate PR) for the more tenured CEs/customer train the trainer as we get there.

@IAmHughes Assuming that you agree with the above, would you mind approving the PR?